config/default.json

```diff
@@ -1,10 +1,114 @@
 {
   "blockchain": {
-    "peerAddress": "localhost:7051",
-    "memberssvcAddress": "localhost:7054",
-    "keystorePath": "/keyValStore",
+    "peerAddress": "services-uscentral.skytap.com:8549",
+    "memberssvcAddress": "services-uscentral.skytap.com:8447",
+    "keystorePath": "/Users/Craig/keyValStore",
```

Comment:
Can we set this to just /keyValStore and use __dirname + config.blockchain.keystorePath (project directory/keyValStore) within setup.js?